cluster-test/test-orchestrator/command.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+var ringpopPort = "3000"
+
+// Command runs command that affect the cluster in different ways. Commands are
+// commenly used to form the Script field of the Scenario struct.
+type Command struct {
+	// Indicates when the command is run.
+	Label string
+
+	// Cmd can be one of:
+	// - `cluster-kill`
+	// - `cluster-start`
+	// - `cluster-rolling-restart`
+	// - `network-drop <SPLIT> <PERCENTAGE>`
+	// - `network-delay <SPLIT> <DURATION>`
+	// - `wait-for-stable`
+	Cmd string
```

Comment:
Also, this just holds the information after the parsing (other pr) so it doesn't contain information if the commands that are passed make sense or have the correct number of arguments.